app/config/FrontendAppConfig.scala

```diff
@@ -62,6 +62,8 @@ trait AppConfig extends ServicesConfig {
 class FrontendAppConfig @Inject()(val runModeConfiguration: Configuration, environment: Environment) extends AppConfig {
   override protected def mode: Mode = environment.mode
 
+  lazy val appName: String = runModeConfiguration.getString("appName").getOrElse(throw new Exception("Missing configuration key: appName"))
```

Comment:
How come you're using this way of getting the appName rather than using getString with a config key? getString will throw an exception if the config can't be found as well.